src/stactools/modis/metadata.py

```diff
@@ -13,6 +14,31 @@
 from stactools.modis import utils
 from stactools.modis.constants import TEMPORALLY_WEIGHTED_PRODUCTS
 
+SIN_SPHERE_RADIUS = 6371007.181
+SIN_TILE_METERS = 1111950
+SIN_X_MIN = -20015109
+SIN_Y_MAX = 10007555
```

Comment:
Is there a document that we can link to, in a comment, pointing to where these values came from?

Reply:
Yep. Done.